pom.xml

```diff
@@ -80,11 +80,11 @@
       <repository>
          <id>eclipse.platform.4.24</id>                                 <layout>p2</layout>
          
-         <url>https://download.eclipse.org/eclipse/updates/4.24-I-builds/I20220525-1800/</url>
 <!-- 
+         <url>https://download.eclipse.org/eclipse/updates/4.24-I-builds/I20220525-1800/</url>
```

Comment:
@wolfgang-ch Note that, I commented your nightly build eclipse because, for some reason that I don't understand, if I don't use the official release, the UI tests build fails with that:

```
2022-06-02T21:54:27.3053237Z [ERROR] Cannot resolve project dependencies:
2022-06-02T21:54:27.3143533Z [ERROR]   Software being installed: net.tourbook.ui.tests 22.6.0.qualifier
2022-06-02T21:54:27.3145372Z [ERROR]   Missing requirement: net.tourbook.ui.tests 22.6.0.qualifier requires 'osgi.bundle; org.eclipse.swtbot.go 3.1.0' but it could not be found
2022-06-02T21:54:27.3146384Z [ERROR] 
2022-06-02T21:54:27.3147443Z [ERROR] See https://wiki.eclipse.org/Tycho/Dependency_Resolution_Troubleshooting for help.
2022-06-02T21:54:27.3292618Z [ERROR] Cannot resolve dependencies of MavenProject: net.tourbook:net.tourbook.ui.tests:22.6.0-SNAPSHOT @ D:\a\mytourbook-BUILD-autocreated\core\net.tourbook.ui.tests\pom.xml: See log for details -> [Help 1]
```

Reply:
I review your PR's after MT 22.6 is released which will also contain the released version of Eclipse in the pom file

Reply:
That sounds good